src/highlight/highlightUtil.ts

```diff
@@ -22,30 +27,41 @@ export function getHighlightArea(shapes: Rect[]): number {
 
 export function getRange(): Range | null {
     const selection = window.getSelection();
-    if (!selection || selection.isCollapsed) {
+    if (!selection || selection.type !== 'Range' || selection.isCollapsed) {
         return null;
     }
 
     return selection.getRangeAt(0);
 }
 
+export const DOMRectToRect = ({ height, width, x, y }: DOMRect): Rect => ({
```

Comment:
This is using constructor naming convention. Can we use camelCase? Rather than using the annotation-specific Rect type, can we use `Pick<DOMRect, 'height' | 'width' | 'x' | 'y'>`?